okhttp/src/main/java/okhttp3/internal/connection/RealConnection.kt

```diff
@@ -690,7 +690,7 @@ class RealConnection(
       if (e is StreamResetException) {
         when (e.errorCode) {
           ErrorCode.REFUSED_STREAM -> {
-            // Retry REFUSED_STREAM errors once on the same connection.
+            // Stop using this connection after one REFUSED_STREAM error.
```

Comment:
two?

Reply:
Yeah, that doesn’t read as well as I wanted. Fixed:

```
            // Stop using this connection on the 2nd REFUSED_STREAM error.
```